solutions/standard_solutions/MediaOps/Tutorial_Scheduling_Encoder_Decoder.md

```diff
@@ -0,0 +1,142 @@
+---
+uid: Tutorial_MediaOps_Scheduling_Encoder_Decoder
+---
+
+# Getting started with the Scheduling app
+
+In this tutorial, you will learn how to create and schedule jobs, swap resources and confirm the created jobs.
+
+Expected duration: 15 minutes
+
+> [!NOTE]  
+> The content and screenshots for this tutorial have been created using DataMiner version 10.5.5 and MediaOps version 1.2.5-CU0.
+
+> [!TIP]  
+> See also: [Scheduling documentation](xref:MO_Scheduling)
+
+## Prerequisites
+
+- A DataMiner System using DataMiner 10.5.5 or higher.
+- Access to the **Resource Studio** application.
+- Having some reasources ready to be assigned to jobs. You can find out how to create those in the [Resource Studio tutorial](xref:Tutorial_MediaOps_Resource_Studio).
+
+## Overview
+
+- [Step 1: Create a new job](#step-1-create-a-new-job)
+- [Step 2: Add resources to the job](#step-2-add-resources-to-the-job)
+- [Step 3: Reserving the resource](#step-3-reserving-the-resource)
+- [Step 4: Add another resource](#step-4-add-another-resource)
+- [Step 5: Swap resources](#step-5-swap-resources)
+- [Step 6: Confirming the job](#step-6-confirming-the-job)
+
+## Step 1: Create a new job
+
+1. Head over to the new job panel by clicking the `+ New` button.
+
+    ![NewJob Button Placement](~/solutions/images/Scheduling_New_Job_Button_Placement.png)
+
+1. Give the job a name of your choice and `Create` it (There is no need to change any other default values for now.)
```

Comment:
I would add to set the start time in a couple of hours to give time before the job actually starts